### PR TITLE
add thinder ha

### PR DESCRIPTION
**Pull request by jzhupup**

### CODE REVIEW

**Comment by jzhupup** [comment]
需要研发进行review 后才能合并

File: docs/zh/docs/install/best-practices/thinder-ha.md [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+
+1. 验证 dnsServer 解析是否正确
+
+  ```bash
+  nslookup www.tinder-node-server.com xxx.xx.xx.192
+  ```
+
+  ![dnsserver](../images/dnsserver01.png)
+
+!!!note
```
Comment (windsonsea) [inline docs/zh/docs/install/best-practices/thinder-ha.md:100]:
```suggestion
!!! note
```

File: docs/zh/docs/install/best-practices/thinder-ha.md [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+
+   - 如果使用外部dns服务解析域名时，需要确保各个节点包括火种节点的/etc/hosts文件中不能出现该域名配置
```
Comment (windsonsea) [inline docs/zh/docs/install/best-practices/thinder-ha.md:102]:
```suggestion
    - 如果使用外部dns服务解析域名时，需要确保各个节点包括火种节点的/etc/hosts文件中不能出现该域名配置
```

File: docs/zh/docs/install/best-practices/thinder-ha.md [hunk 3]
```diff
@@ -0,0 +1,263 @@
...
+
+   - 可以使用nslookup命令检查域名解析状况，需要保证各个节点包括火种节点中，都能使用nslookup来检验解析状况
```
Comment (windsonsea) [inline docs/zh/docs/install/best-practices/thinder-ha.md:104]:
```suggestion
    - 可以使用nslookup命令检查域名解析状况，需要保证各个节点包括火种节点中，都能使用nslookup来检验解析状况
```

File: docs/zh/docs/install/best-practices/thinder-ha.md [hunk 4]
```diff
@@ -0,0 +1,263 @@
...
+
+整体架构如下
+
+![dce01](../images/dce01.png)
+
+1. 安装流程请参考[离线安装 DCE 5.0 商业版](../commercial/start-install.md)
+
+1. 定义好的 clusterConfig 文件参考：
+
+  ```yaml
```
Comment (windsonsea) [inline docs/zh/docs/install/best-practices/thinder-ha.md:116]:
```suggestion
    ```yaml
```
下面的缩进都要同步修改

File: docs/zh/docs/install/best-practices/thinder-ha.md [hunk 5]
```diff
@@ -0,0 +1,263 @@
...
+    addonPackage:
+    binaries:
+      type: builtin
+  ```
+
+1. 开始安装
+
+  ```bash
+  ./dce5-installer cluster-create -c sample/clusterConfig.yaml -m sample/manifest.yaml
+  ```
```
Comment (windsonsea) [inline docs/zh/docs/install/best-practices/thinder-ha.md:150]:
```suggestion
    ```bash
    ./dce5-installer cluster-create -c sample/clusterConfig.yaml -m sample/manifest.yaml
    ```
```

**Review by windsonsea — COMMENTED** [review]
The [preview](https://deploy-preview-3673--daocloud-docs.netlify.app/install/best-practices/thinder-ha) looks not good.

**Review by windsonsea — APPROVED** [review]

**Pull request merged**